log/log_test.go

```diff
@@ -0,0 +1,84 @@
+package log
```

Comment:
@kwk I test things inside of this package, I don't see why this "accident" should happen. But if you start to test things that might cause this accident. Then I'd be okay with doing it.

In these tests, I didn't intend to create a blackbox test.